specification/authorization/resource-manager/Microsoft.Authorization/preview/2015-07-01/authorization-ElevateAccessCalls.json

```diff
@@ -0,0 +1,71 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "AuthorizationManagementClient",
+    "version": "2015-07-01",
+    "description": "Role based access control provides you a way to apply granular level policy administration down to individual resources or resource groups. These operations enable you to manage role definitions and role assignments. A role definition describes the set of actions that can be performed on resources. A role assignment grants access to Azure Active Directory users."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/providers/Microsoft.Authorization/elevateAccess": {
+      "post": {
+        "tags": [
+          "ElevateAccess"
+        ],
+        "operationId": "ElevateAccess_Post",
```

Comment:
This generates a client named `ElevateAccess` with a method named `Post` which isn't very descriptive.  What about something like `Access_Elevate`, or perhaps `GlobalAdministrator_ElevateAccess`?  The second is a little verbose however if there will be other operations for non-global admins then perhaps it makes sense.

Reply:
I think `GlobalAdministrator_ElevateAccess` follows closer to `ClassicAdministrators_List` which we already have, so I will change it to that.